lib/get-client.js

```diff
@@ -1,62 +1,42 @@
-const {memoize, get} = require('lodash');
 const {Octokit} = require('@octokit/rest');
-const pRetry = require('p-retry');
-const Bottleneck = require('bottleneck');
+const {throttling} = require('@octokit/plugin-throttling');
+const {retry} = require('@octokit/plugin-retry');
 const urljoin = require('url-join');
 const HttpProxyAgent = require('http-proxy-agent');
 const HttpsProxyAgent = require('https-proxy-agent');
 
-const {RETRY_CONF, RATE_LIMITS, GLOBAL_RATE_LIMIT} = require('./definitions/rate-limit');
+const SemanticReleaseOctokit = Octokit.plugin(throttling, retry);
 
-/**
- * Http error status for which to not retry.
- */
-const SKIP_RETRY_CODES = new Set([400, 401, 403]);
-
-/**
- * Create or retrieve the throttler function for a given rate limit group.
- *
- * @param {Array} rate The rate limit group.
- * @param {String} limit The rate limits per API endpoints.
- * @param {Bottleneck} globalThrottler The global throttler.
- *
- * @return {Bottleneck} The throller function for the given rate limit group.
- */
-const getThrottler = memoize((rate, globalThrottler) =>
-  new Bottleneck({minTime: get(RATE_LIMITS, rate)}).chain(globalThrottler)
-);
+const {RETRY_CONF} = require('./definitions/rate-limit');
 
 module.exports = ({githubToken, githubUrl, githubApiPathPrefix, proxy}) => {
   const baseUrl = githubUrl && urljoin(githubUrl, githubApiPathPrefix);
-  const globalThrottler = new Bottleneck({minTime: GLOBAL_RATE_LIMIT});
-  const github = new Octokit({
+  const github = new SemanticReleaseOctokit({
     auth: `token ${githubToken}`,
     baseUrl,
     request: {
+      retries: RETRY_CONF.retries,
```

Comment:
why did you set that? I don't think that is right, it's set dynamically by the throttling plugin.

Removing it fixes the "Throw SemanticReleaseError for invalid token" error

Reply:
I think other tests were failing when I did not set that. Tbh, it was through [denial and error](https://youtu.be/Jfq3c4Cf1Fs?t=68).